README.md

```diff
@@ -45,42 +45,42 @@ Lets walk through common flow for http request:
     actionPipe.send(new SampleHttpAction());
     // or actionPipe.createObservable(new SampleHttpAction()).subscribe(...) 
     ```
-2. Request is forwarded to `Janet` instance upon `send` or `subscribe` call;
-3. `Janet` finds suitable service to execute action and routes to it;
-4. `ActionService` knows how to deal with action and sets up `progress`/`success`/`fail` statuses;
-5. Resulting action is brought back to `Janet` instance;
-6. `Janet` routes resulting action with status to dedicated pipes;
-7. Dedicated `ActionPipe` notifies all it's observers of action wrapped with current status;
+2. The request is forwarded to the `Janet` instance upon the `send` or `subscribe` call;
+3. `Janet` finds a suitable service to execute an action and routes to it;
+4. `ActionService` knows how to deal with the action and sets up the `progress`/`success`/`fail` statuses;
+5. The resulting action is brought back to the `Janet` instance;
+6. `Janet` routes the resulting action with a status to the dedicated pipes;
+7. The dedicated `ActionPipe` notifies all its observers of the action wrapped with a current status;
 
-So `Janet` instance itself stands for routing, delegating diff job to other components:
+So the `Janet` instance itself stands for routing and delegating the diff job to other components:
 
-* `ActionPipe` – action operator, the only way to send action and receive result;
-* `ActionService` – knows how to deal with action;
-* `ActionServiceWrapper` – decorator to put additional logic for underlying service;
+* `ActionPipe` – an action operator, the only way to send an action and receive a result;
+* `ActionService` – knows how to deal with an action;
+* `ActionServiceWrapper` – a decorator to add additional logic to the underlying service;
 
 ### Available services
-Janet abilities depends on services.
-Currently there are:
+The Janet abilities depend on services.
+Currently, they include the following:
 
-* [HttpActionService](https://github.com/techery/janet-http) to provide HTTP/HTTPS requests execution;
+* [HttpActionService](https://github.com/techery/janet-http) to enable the HTTP/HTTPS request execution;
 * [AsyncActionService](https://github.com/techery/janet-async) to provide support for async protocols, e.g. [socket.io](http://socket.io/);
-* [CommandActionService](https://github.com/techery/janet-command) to delegate job back to command `action`.
+* [CommandActionService](https://github.com/techery/janet-command) to delegate the job back to the command `action`.

```

Comment:
could we use `any job` instead of `the job`?

Reply:
sure we can, I'll change it